recipes/openssl/3.x.x/conanfile.py

```diff
@@ -712,7 +712,7 @@ def _module_file_rel_path(self):
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "OpenSSL")
-        self.cpp_info.set_property("cmake_target_name", "OpenSSL")
+        self.cpp_info.set_property("cmake_find_mode", "both")
```

Comment:
And what kind of CMake variables names can we expect in the find file?

Reply:
Yes, that won't work properly until we fix the variable names problem in the cmake modules that cmakedeps generates.